tasks/main.yml

```diff
@@ -1,22 +1,9 @@
 ---
-- name: Clone Certbot into configured directory.
-  git:
-    repo: "{{ certbot_repo }}"
-    dest: "{{ certbot_dir }}"
-    version: "{{ certbot_version }}"
-    update: "{{ certbot_keep_updated }}"
-    force: yes
+- include: install-from-package.yml
+  when: not certbot_from_git and (ansible_distribution == 'Fedora' or ansible_distribution == 'RedHat')
```

Comment:
I'll want to support package install on Debian/Ubuntu as well, before a merge. I can work on that if you like.